packages/salesforcedx-vscode-core/test/jest/commands/delete/forceSourceDelete.test.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import { ForceSourceDeleteExecutor } from '../../../../src/commands';
+
+describe('Force Source Delete', () => {
+  let isSourceTracked: boolean;
+  const sourcePath: string = path.join('example', 'path');
+  it('Should build the source delete command with --tracksource flag', () => {
```

Comment:
nit: can you put an empty line before each test to make the file a bit easier to read?